lib/staleBuildChecks.js

```diff
@@ -91,6 +64,21 @@ const removeOldBuildLabel = async (context) => {
     return;
   }
 
+  // Get last commit for pull request.
+  const {data: lastCommit} = await context.github.repos.getCommit(
+    context.repo({
+      ref: pullRequest.head.sha
+    })
+  );
+
+  const lastCommitDate = new Date(lastCommit.commit.author.date);
+  // Commit is older than 2 days
```

Comment:
```suggestion
  // Commit is older than 2 days, this can happen when 'synchronize' event 
  // got triggered as a result of a merge to the main branch. In this case
  // we don't want to remove the label.
```